### ISSUE TITLE
Na busca avançada, selecionar tipo de post

### ISSUE THREAD

**Issue by jacsonp** [redelivre/flacso#141]
Poder selecionar se a busca será por post e publication

**Comment by eduardozulian** [comment]
Isso foi pedido? Tem a ver com as novas questões sobre as publicações internas e externas?

**Comment by jacsonp** [comment]
Não, apenas uma ideia que surgiu após discussão sobre separação de conteúdo que tivemos
